cloud_utils/cache/stores/pickle_store.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import os
+import pickle
+import time
+from typing import Any, Callable, Dict, Tuple
+
+import gamla
+
+
+def _local_cache_path(cache_name: str, path: str):
+    return os.path.join(path, f"{cache_name}.pickle")
+
+
+@gamla.timeit
+def _load_cache_from_local(cache_name: str, path: str) -> Dict[Tuple, Any]:
+    with open(_local_cache_path(cache_name, path), "rb") as local_cache_file:
+        return pickle.load(local_cache_file)
+
+
+def _save_cache_locally(cache_name: str, path: str, cache: Dict[Tuple, Any]):
+    if not os.path.exists(path):
+        os.makedirs(path)
+    local_path = _local_cache_path(cache_name, path)
+    with open(local_path, "wb") as local_cache_file:
+        pickle.dump(cache, local_cache_file)
+    logging.info(f"Saved {len(cache)} cache items locally for {cache_name}.")
+
+
+def make_store(name: str, cache_path: str) -> Tuple[Callable, Callable]:
```

Comment:
add the initialize log here

Reply:
There is initialising logs there in both try an except